test/trento/domain/cluster/cluster_test.exs

```diff
@@ -923,6 +922,15 @@ defmodule Trento.ClusterTest do
         assert match?({:error, :cluster_not_registered}, aggregate_run(initial_events, command)),
                "Command #{inspect(command)} should be rejected by the aggregate"
       end
+
+      commands_to_accept = [
+        %RollUpCluster{}
```

Comment:
Here we could include the `RegisterClusterHost` command as well, for completeness